instrument/HB3A_Definition_2017-04-25.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='ASCII'?>
+<instrument xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.mantidproject.org/IDF/1.0" last-modified="2017-05-08 10:35:21.877396" name="HB3A" valid-from="2017-04-25 00:00:00" valid-to="2100-12-31 23:59:59" xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd">
```

Comment:
Don't use midnight as a valid time. One minute or second before or after is less confusing.